readthedocs/projects/forms.py

```diff
@@ -634,6 +634,23 @@ def clean_domain(self):
             domain_string = parsed.netloc
         else:
             domain_string = parsed.path
+
+        if (
+            settings.PRODUCTION_DOMAIN and
+            domain_string.endswith(settings.PRODUCTION_DOMAIN)
+        ):
+            raise forms.ValidationError(
+                f'{settings.PRODUCTION_DOMAIN} is not a valid domain.'
+            )
+
+        if (
+            settings.PUBLIC_DOMAIN and
+            domain_string.endswith(settings.PUBLIC_DOMAIN)
+        ):
+            raise forms.ValidationError(
+                f'{settings.PUBLIC_DOMAIN} is not a valid domain.'
+            )
```

Comment:
This can be done in a for and de-duplicate this code, right?

Reply:
Okay :+1: